### ISSUE TITLE
Compile failed

### ISSUE THREAD

**Issue by wangbo4020** [SunnyApp/flutter_contact#61]
run ``` flutter pub upgrade``` then ```flutter build apk --profile --flavor offical```

flutter_contact: 0.8.1+2
Flutter Version: Any version.

```log

/C:/Users/Dylan/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_contact-0.8.1+2/lib/contact.dart:306:12: Error: The argument type 'Iterable<Item>' can't be assigned to the parameter type 'List<Item>?'.
 - 'Iterable' is from 'dart:core'.
 - 'Item' is from 'package:flutter_contact/contact.dart' ('/C:/Users/Dylan/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_contact-0.8.1+2/lib/contact.dart').
 - 'List' is from 'dart:core'.
          .notNull(),
           ^
/C:/Users/Dylan/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_contact-0.8.1+2/lib/contact.dart:308:12: Error: The argument type 'Iterable<Item>' can't be assigned to the parameter type 'List<Item>?'.
 - 'Iterable' is from 'dart:core'.
 - 'Item' is from 'package:flutter_contact/contact.dart' ('/C:/Users/Dylan/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_contact-0.8.1+2/lib/contact.dart').
 - 'List' is from 'dart:core'.
          .notNull(),
           ^
/C:/Users/Dylan/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_contact-0.8.1+2/lib/contact.dart:313:12: Error: The argument type 'Iterable<Item>' can't be assigned to the parameter type 'List<Item>?'.
 - 'Iterable' is from 'dart:core'.
 - 'Item' is from 'package:flutter_contact/contact.dart' ('/C:/Users/Dylan/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_contact-0.8.1+2/lib/contact.dart').
 - 'List' is from 'dart:core'.
          .notNull(),
           ^
/C:/Users/Dylan/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_contact-0.8.1+2/lib/contact.dart:316:9: Error: The argument type 'Iterable<ContactDate>' can't be assigned to the parameter type 'List<ContactDate>?'.
 - 'Iterable' is from 'dart:core'.
 - 'ContactDate' is from 'package:flutter_contact/contact.dart' ('/C:/Users/Dylan/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_contact-0.8.1+2/lib/contact.dart').
 - 'List' is from 'dart:core'.
      ].notNull(),
        ^
/C:/Users/Dylan/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_contact-0.8.1+2/lib/contact.dart:320:9: Error: The argument type 'Iterable<PostalAddress>' can't be assigned to the parameter type 'List<PostalAddress>?'.
 - 'Iterable' is from 'dart:core'.
 - 'PostalAddress' is from 'package:flutter_contact/contact.dart' ('/C:/Users/Dylan/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_contact-0.8.1+2/lib/contact.dart').
 - 'List' is from 'dart:core'.
      ].notNull(),
        ^
/C:/Users/Dylan/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/flutter_contact-0.8.1+2/lib/contact.dart:561:7: Error: A value of type 'Iterable<Map<String, String>>' can't be returned from a function with return type 'List<Map<Stri
ng, String>>'.
 - 'Iterable' is from 'dart:core'.
 - 'Map' is from 'dart:core'.
 - 'List' is from 'dart:core'.
    ].notNull();
      ^


FAILURE: Build failed with an exception.

* Where:
Script 'D:\Development\Flutter\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 1035

* What went wrong:
Execution failed for task ':app:compileFlutterBuildOfficalProfile'.
> Process 'command 'D:\Development\Flutter\flutter\bin\flutter.bat'' finished with non-zero exit value 1


```

**Comment by wangbo4020** [comment]
If I want run, should write under code to pubspec.yaml

```yaml
dependency_overrides:
  dartxx: 0.4.4+6
  flexidate: 0.4.5-nullsafety.2
```

**Comment by ericmartineau** [comment]
Fixed  `^0.9.1+5`
